editor/js/libs/app.js

```diff
@@ -189,6 +204,10 @@ var APP = {
 
 		this.stop = function () {
 
+			if ( vrEnabled() ) {
+				vrButton.remove();
```

Comment:
For consistency...
```js
document.body.remove( vrButton );
```

Reply:
I tried this, but calling document.body.remove detaches the html `<body>` element from the dom, even if another element is passed in as an argument.